app/models/greeting.rb

```diff
@@ -0,0 +1,2 @@
+class Greeting < ApplicationRecord
```

Comment:
- [ ] I also noticed that in your schema and seed file, you had `Message ` but here in your model, you have `Greeting` which does not correlate. Kindly note that the naming convention was supposed to match from the controller to the model and to the database tale.